jgrapht-core/src/main/java/org/jgrapht/graph/SynchronizedGraphParams.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * (C) Copyright 2018-2018, by CHEN Kui and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.graph;
+
+/**
+ * The SynchronizedGraph Params describes the properties of a AsSynchronizedGraph to be created such
+ * as whether it uses cache.
+ *
+ * @author CHEN Kui
+ * @since Mar 9, 2018
+ */
+public class SynchronizedGraphParams
+{
+    private boolean cacheEnable;
+
+    /**
+     * Constructor for SynchronizedGraphParams with cacheDisable.
+     */
+    public SynchronizedGraphParams()
+    {
+        cacheEnable = false;
+    }
+
+    /**
+     * Make the AsSynchronizedGraph to be created <strong>not</strong> use cache for <code>edgesOf</code>,
```

Comment:
"Request a synchronized graph without caching."  (If they want the details of which methods it applies to, they can read the AsSynchronizedGraph Javadoc.)